<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsolePropertiesClass</class>
 <widget class="QWidget" name="ConsolePropertiesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConsoleProperties</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Terminal</string>
   </property>
   <widget class="QComboBox" name="deviceCombo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1052</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3215</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1052-C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3215-C</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceNumber">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>78</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>78</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_terminal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminal Group</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="terminalGroup">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandPrefix">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>x; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipAddress">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hhh.hhh.hhh.hhh; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mask">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Subnet Mask</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subnetMask">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hhh.hhh.hhh.hhh; </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="noPromptCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>No Prompt</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
